ModList/UI/Components/MarkdownText.cs

```diff
@@ -283,20 +294,20 @@ public void OnPointerEnter(PointerEventData eventData)
 
             public void OnPointerExit(PointerEventData eventData)
                 => Manager.EndHover();
+
             public void OnPointerClick(PointerEventData eventData)
                 => Manager.Click();
 
             protected override void UpdateMaterial() { }
             protected override void UpdateGeometry() { }
-
         }
 
         private class LinkHoverHint : HoverHint
         {
             public HoverHintController Controller
             {
                 get => _hoverHintController;
-                set => _hoverHintController = value;
+                set => this.SetField<HoverHint, HoverHintController>("_hoverHintController", value);
```

Comment:
I take it the field was made readonly? Do we need to actually set the field most of the time?

Also, `nameof`, please.

Reply:
It was indeed marked as readonly, hence the use of  `.SetField<T, U>()`.
That being said, I simply fixed this as it prevented the code from compiling, but I honestly don't know whether this is actually being used at all...

Forgot to use `nameof ()` so will fix that.